components/buttons/buttons.css

```diff
@@ -0,0 +1,66 @@
+.btn{
+    border-radius:0.5rem;
+    padding:0.3rem 0.3rem 0.3rem 0.3 rem;
+    margin:0.3rem 0.3rem 0.3rem 0.3rem;
+    height:auto;
+    width:auto;
+    display: inline-block;
+    cursor: pointer;
+}
+
+.primary-btn{
+    background-color: var(--primary-color);
+    color: black;
+}
+
+.secondary-btn{
+    background-color: var(--secondary-color);
+    color: var(--text-gray);
+}
+
+.disabled-btn{
+    pointer-events: none;
+    color: var(--text-lighter);
+    background-color: var(--light-gray);
+}
+
+.primary-btn-outlined{
+    background-color: white;
+    color: var(--primary-color);
+    border: 0.2rem solid var(--primary-color);
+}
+
+.secondary-btn-outlined{
+    background-color: white;
```

Comment:
Use variable for white color too

Reply:
Hi Abhishek
thanks for the review. Changes suggested by you aniket have been incorporated now

Reply:
@abhisheksahani-nova Could you please approve the PR as well?

Reply:
Yess go ahead